container_pipeline/workers/build.py

```diff
@@ -35,11 +35,11 @@ def handle_job(self, job):
                 is_build_running
 
         if parent_build_running:
-            self.logger.info('Parents in build: {}, pushing job: {} back '
+            self.logger.debug('Parents in build: {}, pushing job: {} back '
```

Comment:
The way I've differentiated between `info` and `debug` is that whenever we're storing some dynamic data into the logs, it goes under `debug`. In this case, we're logging the data about `parents_in_build` and `job` which we get to know dynamically only. Because, as a user I'm not really going to make much from that data. But an admin/dev would find that information helpful.